From c74c3a5cecc3e05a9bea3d7362a4b784b668a7da Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 08:19:30 +0300
Subject: [PATCH 39/50] fdts: s32cc: Correct QSPI simple_bus_reg warning

This corrects dtc warning:
Warning (simple_bus_reg): /soc/spi@40134000: simple-bus unit address
format error, expected "0"

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 56fbd1e5f..85f8c6689 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -503,9 +503,9 @@
 			compatible = "nxp,s32cc-qspi";
 			#address-cells = <1>;
 			#size-cells = <0>;
-			reg = <0x0 0x00000000 0x0 0x20000000>,
-				<0x0 0x40134000 0x0 0x1000>;
-			reg-names = "QuadSPI-memory", "QuadSPI";
+			reg = <0x0 0x40134000 0x0 0x1000>,
+			      <0x0 0x00000000 0x0 0x20000000>;
+			reg-names = "QuadSPI", "QuadSPI-memory";
 			clock-names = "qspi_en", "qspi";
 			clocks = <&clks S32GEN1_SCMI_CLK_QSPI_FLASH1X>,
 				 <&clks S32GEN1_SCMI_CLK_QSPI_FLASH1X>;
-- 
2.17.1

